staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/webhook_converter.go

```diff
@@ -86,7 +86,7 @@ func webhookClientConfigForCRD(crd *internal.CustomResourceDefinition) *webhook.
 		ret.Service = &webhook.ClientConfigService{
 			Name:      apiConfig.Service.Name,
 			Namespace: apiConfig.Service.Namespace,
-			Port:      apiConfig.Service.Port,
+			Port:      *apiConfig.Service.Port,
```

Comment:
can this be nil?

Reply:
the defaulting prevents nil pointer for us
```go
	if obj.Port == nil {
		obj.Port = utilpointer.Int32Ptr(443)
	}
```